Updated the name of the role in the example RoleBinding YAML file to match the defined role. (#70057)
openshift/openshift-docs: modules/gitops-additional-permissions-for-cluster-config.adoc

  resources: ["secrets"]
  verbs: ["*"]
----

. Click *Create* to add the cluster role.

> . Now create the cluster role binding. In the main menu, again select *User Management* -> *Role Bindings* -> *Create Binding*.

Reviewer: What is the `main menu`? That phrase is not currently used in the gitops docs.

Author: Hi @kalexand-rh , 
![image](https://github.com/openshift/openshift-docs/assets/99152915/da4d135c-4c88-4f1f-9877-fb4e078377cd)

> . Now create the cluster role binding. In the main menu, again select *User Management* -> *Role Bindings* -> *Create Binding*.

Reviewer: I don't think it's necessary to call out that menu. Maybe:
```suggestion
. To create the cluster role binding, select *User Management* -> *Role Bindings* -> *Create Binding*.
```

Author: ACK.

Addressed
